modules/ROOT/pages/traces.adoc

```diff
@@ -22,6 +22,15 @@ image::traces.png["Span list in Traces Overview"]
 
 In addition to reviewing traces in Anypoint Monitoring, you can use xref:telemetry-exporter.adoc[Telemetry Exporter] to export traces to third-party observability and analytics platforms.
 
+== Traces Availability
```

Comment:
Rationale for section: Collapsing H3s under an H2 bc I don't want 3 H2s for these short sections showing up in the right nav and diluting visibility of other important H2 sections.